src/ProgressOnderwijsUtils/ProgressOnderwijsUtils.csproj

```diff
@@ -1,20 +1,20 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup Label="Configuration">
-    <Version>12.0.0</Version>
+    <Version>13.0.0</Version>
     <Authors>Progress Onderwijs</Authors>
     <Owners>Progress Onderwijs</Owners>
     <Title>ProgressOnderwijsUtils</Title>
     <Description>
       Various utility methods+classes used by Progress Onderwijs.
     </Description>
     <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
-    <PackageReleaseNotes>Removed method RandomHelper.GetPasswordString</PackageReleaseNotes>
+    <PackageReleaseNotes>Split off System.Web and System.Drawing related types and removed net461 target.</PackageReleaseNotes>
     <PackageTags>ProgressOnderwijs</PackageTags>
     <PackageLicenseUrl>http://www.apache.org/licenses/LICENSE-2.0</PackageLicenseUrl>
     <PackageProjectUrl>https://github.com/progressonderwijs/ProgressOnderwijsUtils</PackageProjectUrl>
   </PropertyGroup>
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
```

Comment:
Bij single-targetting: als je dit nou pushed als beta versie, is de dependency chain in bijv. gadgets dan gestoord of niet?

Reply:
(de vraag achter de vraag: is netstandard2.0 net zo onvriendelijk in het gebruik als netstandard1.1 vanuit een old-style project met transitive-closure dependencies, of is dit inmiddels beter?)

Reply:
Gestoord lijkt me subjectief, maar ik vermoed dat het antwoord is dat het nog veel gestoorder is als je niet .NET 4.7.1 gebruikt.

Wel een gepasseerd station, want we dependen nu ook al netstandard2.0-libraries.

Reply:
Hmm, mischien helpt dat ook niet mee met de onwerkbaar trage nuget updates in PNet.  Tja.

Hij pakt de .net framework versies eerst lijkt het, maar ik heb daar geen goede docs voor ooit gevonden.

Reply:
Ik ben er mee bezig: https://github.com/progressonderwijs/progress/pull/13589

Hij heeft in elk geval wel net461 gepakt. Lijkt me ook wel logisch met het oog op backwards compatibility. Volgens mij is er zelfs een override-mogleijkheid.